src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -23,6 +26,7 @@
 
     private Model notablyModel;
     private CorrectionEngine<AbsolutePath> pathCorrectionEngine;
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `OpenCommandParser`!